src/js/components/settings/cardsection.js

```diff
@@ -0,0 +1,115 @@
+/**
+* Use the CSS tab above to style your Element's container.
+*/
```

Comment:
that is probably a remainder from the stripe website...

Reply:
yup